kops/cloud-platforms-test.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: null
+spec:
+  api:
+    loadBalancer:
+      type: Public
+  additionalPolicies:
+    node: |
+      [
+        {
+          "Effect": "Allow",
+          "Action": [
+            "route53:ChangeResourceRecordSets"
+          ],
+          "Resource": [
+            "arn:aws:route53:::hostedzone/Z1N3S2UZYKP74S"
+          ]
+        },
+        {
+          "Effect": "Allow",
+          "Action": [
+            "route53:ListHostedZones",
+            "route53:ListResourceRecordSets"
+          ],
+          "Resource": [
+            "*"
+          ]
+        }
+      ]
+  authorization:
+    rbac: {}
+  channel: stable
+  cloudProvider: aws
+  iam:
+    allowContainerRegistry: true
+    legacy: false
+  kubeAPIServer:
+    oidcClientID: 7ix4YkYwQe8n7j9qTrdAKeMz26eJTB2x
```

Comment:
`oidcClientId` and `oidcIssuerURL` will also vary per cluster, so should be set by the script

Reply:
by create_cluster_config.py?

Reply:
Yep - although, as per the discussion on how we should manage per-cluster config, maybe this actually should be a static value in an individual cluster's YAML file, so I'd leave this for now

Reply:
ok

Reply:
as we determined to have a per cluster yaml file, this will be static for now.